frontend/apps/reader/modules/readerfont.lua

```diff
@@ -791,22 +811,24 @@ function ReaderFont:addToRecentlySelectedList(face)
     end
 end
 
-function ReaderFont:sortFaceList(face_list)
+function ReaderFont:setupFaceList(face_list)
+    self.face_table.needs_refresh = true
+
+    -- Init list of previously known fonts to empty table
```

Comment:
I have a hard time understanding if the refactoring from here until the bottom of the PR diff does change any behaviour, or not. Does it?
And why the changes, just because the current implementation wasn't really obvious, and you felt like rewriting it in a way that was more readable to you (so your added/tweaked comments)?

Reply:
It shouldn't change any behavior; it's factoring out the menu setup. Before, sortFaceList() would detect if it was the first time called after launch and do some setup. I moved that setup to its own function that gets called when launching. Now sortFaceList() just does the sorting.